src/book/mdbook.rs

```diff
@@ -38,7 +38,7 @@ impl MDBook {
                         .set_src(&root.join("src"))
                         .set_dest(&root.join("book"))
                         .to_owned(),
-            renderer: Box::new(HtmlHandlebars::new()),
+            renderer: Box::new(Pandoc::new()),
```

Comment:
This is just temporary to make local testing easier, but curious how we see this playing out long term. Should the mdbook command line support a new argument? Default to using all the available renderers?